kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -665,6 +686,27 @@ public boolean isUpsertDeleteEnabled() {
     return getBoolean(UPSERT_ENABLED_CONFIG) || getBoolean(DELETE_ENABLED_CONFIG);
   }
 
+  public TimePartitioning.Type getTimePartitioningType() {
+    return parseTimePartitioningType(getString(TIME_PARTITIONING_TYPE_CONFIG));
```

Comment:
Since we already have a validator - couldn't we use something like 

`return TimePartitioning.Type.valueOf( getString(TIME_PARTITIONING_TYPE_CONFIG).toUpperCase());`

Reply:
Yeah, we could. I don't see much difference either way; figured it'd be a little bit safer to use the same parsing logic in both places just in case a future modification somehow causes things to slip through the cracks. Do you think there's an advantage to your approach?

Reply:
Noted! I'm fine with being more defensive - we can leave as is :)